app/assets/stylesheets/layout.scss

```diff
@@ -62,6 +62,14 @@ body{
     background: $main-panel-background;
     margin: 1em;
 
+    a:hover{
```

Comment:
Previously it was black for some reason, which, yeah, is nearly illegible with dark blue text.